### PR TITLE
various minor tweaks

### PR DESCRIPTION
**Pull request by FND**
please review commits individually; feel free to reject at will

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,22 +2,21 @@ let util = require("util");
 let fs = require("fs");
 let readFile = util.promisify(fs.readFile);
 
-exports.middleware = (manifestPath, { caching } = {}) => {
-	if(caching === undefined) {
-		caching = (process.env.NODE_ENV === "production");
-	}
+let PRODUCTION = process.env.NODE_ENV === "production";
```
Comment (FND) [inline index.js:5]: Note that this is only evaluated on startup now, but middleware initialization effectively also happens at startup I suppose? (I don't think we want to evaluate this for each request.)

File: index.js [hunk 2]
```diff
@@ -2,22 +2,21 @@ let util = require("util");
...
+	// we do it synchronously because we don't want to await the middleware
+	let manifest = require(manifestPath);
+
+	let assetURL = identifier => lookup(manifest, identifier);
 
 	return async (req, res, next) => {
-		// if caching is disabled, refresh the manifest once per request
-		if(!caching) {
+		if(!caching) { // refresh the manifest once per request
 			manifest = await loadJSON(manifestPath);
```
Comment (FND) [inline index.js:17]: Might as well use `require` here too, given that this should only be used in development mode?

**Review by FND — COMMENTED** [review]
FWIW, personal preferences not included here:
```diff
diff --git a/index.js b/index.js
index 11b471f..b79eb6b 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,7 @@
-let util = require("util");
 let fs = require("fs");
-let readFile = util.promisify(fs.readFile);
+let { promisify } = require("util");
+
+let readFile = promisify(fs.readFile);
 
 let PRODUCTION = process.env.NODE_ENV === "production";
 
@@ -21,8 +22,7 @@ exports.middleware = (manifestPath, { caching = PRODUCTION } = {}) => {
 	};
 };
 
-class ManifestError extends Error {
-}
+class ManifestError extends Error {}
 
 exports.ManifestError = ManifestError;
 
```

**Review by moonglum — APPROVED** [review]
Yep, good work! Thanks for your contribution 🙇

**Pull request merged**